packages/client/docusaurus/docs/javascript/02-guides/04-screensharing.mdx

```diff
@@ -11,6 +11,13 @@ If you want to see the device management API in action, you can check out [the s
 Our Video SDK provides an easy ways to enable screensharing in your application.
 In the following examples, we would learn to control and configure screen sharing.
 
+For a user to be able to share their screen,
+they must have the `screenshare` capability configured for the call they are in.
+
+This can be configured on the dashboard for your call type:
+
+![Screenshot shows screensharing dashboard setting](../assets/02-guides/04-screensharing/screensharing-dashboard.png)
```

Comment:
I think the screenshot and text doesn't match, the text talks about the capability, but the screenshot show a call type setting

Reply:
Updated. @szuperaz Can you please check again?